Adding safety profile png and model for Requirement Class (#1061)
spdx/spdx-3-model: model/FunctionalSafety/Properties/requirementHierachyLevel.md

SPDX-License-Identifier: Community-Spec-1.0

> # requirementHierachyLevel

Reviewer: ```suggestion
# requirementHierarchyLevel
```

Need to rename the file as well

## Summary

> Number indicating the hierachical level of the requirement whithin its namespace

Reviewer: ```suggestion
Number indicating the hierarchical level of the requirement within its namespace.
```

## Description

The level of the requirement within the project that gave the requirement its namespace, using a positive number, e.g. in Zephyr: 1 for the Zephyr Safety Claims, 2 for Zephyr system requirements and 3 for Zephyr Software requirements 

> 

Reviewer: ```suggestion
```
